Error in the typescript definitions with noImplictAny

```
jsonschema/lib/index.d.ts(14,5): error TS7010: 'validate', which lacks return-type annotation, implicitly has an 'any' return type.
```

I hope i will get to it and make a PR

--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -11,7 +11,7 @@ export declare class Validator {
     attributes: {[property:string]: CustomProperty};
 
     addSchema(schema?: Schema, uri?: string): Schema|void;
-    validate(instance: any, schema: Schema, options?: Options, ctx?: SchemaContext);
+    validate(instance: any, schema: Schema, options?: Options, ctx?: SchemaContext): ValidatorResult;
 }
 
 export declare class ValidatorResult {

